### ISSUE TITLE
Can you modify allure xml results during runtime?

### ISSUE THREAD

**Issue by paxtonjf** [allure-framework/allure1#976]
I am developing a `protractor` framework using `TypeScript/JavaScript`.
I am using `jasmine` and `allure` for test cases and reports.

As of now, my framework writes `allure` xml files after the automation is finished. Once, automation is finished, I have a process that walks through the `allure-reports` directory and adds a couple tags/values at the end of each `allure` xml file.

I would like to know if the `allure reporter` is able to write those tags at the end of the `allure` xml files during automation (on the fly).

Right now I have:
```
onPrepare: () => {
     var AllureReporter = require('jasmine-allure-reporter');
            jasmine.getEnv().addReporter(new AllureReporter({
                resultsDir : '../../../allure-results'
            }));
        }
```
Basically, I would like to know if there is any way I can write the tags in the `onPrepare()` function, instead of having a separate process at the end write the tags to the `allure` xml files.

Thanks.
